src/base/QXmppOmemoData.cpp

```diff
@@ -24,20 +25,10 @@
 /// \since QXmpp 1.5
```

Comment:
it's probably a good idea to remove the `\since QXmpp 1.5` since it's not part of the public API